### PR TITLE
Refact Content Loaders to work on Firefox

### PR DESCRIPTION
**Pull request by estacioneto**
#### What is the purpose of this pull request?
On Firefox, it was not possible to determine the SVG's style via CSS/Style prop. So, what works on Chrome and Safari, doesn't work on Firefox. To work in all browsers, the found solution was pass every style via elements properties.
In `ProductPrice`, for example, we have:
```jsx
if ((showListPrice && isNil(listPrice)) || isNil(sellingPrice)) {
   return <Price.Loader {...this.props.styles} />
}
```

Inside `Price.Loader`:
```jsx
<rect
   height="0.75em"
   width="50%"
   x="25%"
   {...loaderProps['vtex-price-list__container--loader']}
   />
```

Therefore, components like `ProductDetails` renders `ProductPrice` this way:
```jsx
<ProductPrice
  styles={productPriceLoaderStyles}
  listPrice={path(['ListPrice'], this.commertialOffer)}
  {/* Price props */}
  />
```

And:
```jsx
const productPriceLoaderStyles = {
  'vtex-price-list__container--loader': {
    x: 0,
    width: '7.219em',
    height: '0.56em',
  },
  /* ... */
}
```

Summing up, what was done via CSS classes, now will be done via JSX.

#### What problem is this solving?
Content Loaders not rendering on Firefox.

#### How should this be manually tested?

[Access the workspace](https://estacio--storecomponents.myvtex.com/)

#### Screenshots or example usage

image

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ProductImages/index.js [hunk 1]
```diff
@@ -38,8 +38,42 @@ class ProductImages extends Component {
...
   }
 
-  static Loader = props =>
-    props.isVertical ? (
+  static Loader = (loaderProps = {}) => {
+    const styles = {
+      'vtex-product-image__selected-image--loader': {
+        x: '10%',
+        y: '5%',
+        'max-width': '19.782em',
```
Comment (lucasecdb) [inline react/components/ProductImages/index.js]: `maxWidth`

Comment (lucasecdb) [reply react/components/ProductImages/index.js]: Also, use the `rem` unit, not `em`

Comment (estacioneto) [reply react/components/ProductImages/index.js]: Actually, this property doesn't work. It's not necessary.
